app/schemas/com.hyperether.getgoing.repository.room.AppDatabase/3.json

```diff
@@ -2,11 +2,11 @@
   "formatVersion": 1,
   "database": {
     "version": 3,
-    "identityHash": "12920c91e53a45024bda36cecaaefb8b",
```

Comment:
@zukovicfilip is it safe to change this json? Is app update working properly?

Reply:
Yes

Reply:
@zukovicfilip no, it does not. Install app from Google Play, save some data, then install this as update. You will get error: 
Caused by: java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the version number.

Reply:
@zukovicfilip what is with 3.json ? Is it needed to be changedd if we added 4.json?